src/hooks/services/useAuthenticationService.ts

```diff
@@ -19,7 +19,7 @@ export const useAuthenticationService = () => {
     const register = useCallback(
         async (registerDto: RegisterDto): Promise<string> => {
             try {
-                const response = await authenticationApi.apiAuthenticationRegisterPost(registerDto);
+                const response = await authenticationApi.register(registerDto);
```

Comment:
Looks much better with the method names from open api :)

Reply:
Yes!